.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,54 @@
+name: Docker
+
+on:
+  push:
+    # Publish `master` as Docker `latest` image.
+    branches:
+      - master
```

Comment:
Sorry, one more question: since this seems to explicitly say to only publish an image on pushes to master, why is there already an example image from this branch? Did you upload that one manually, or will it in future actually do it automatically for all branches?

Reply:
No worries, it's useful to know what parts of this whole kerfuffle are unclear.

That was an unfortunate timing mistake. I added this branch to check whether git-hub pipelines were properly working, and then I reverted the commit to build containers only for master (which ought to be the only branch used for production, hence the only branch for which a container is required).

Reply:
That's actually a fortunate timing mistake then, since it told us it's already working. :)